src/app/page.tsx

```diff
@@ -1,108 +1,114 @@
+          <p className="f-size-24">Nuestros números hablan</p>
+          <Link href='' className="bg-primary white-text ph-20 p-10 radius gap-15 flex align-center" style={{width: "60%"}}>
+                Cotizar
+                <i className="fa-solid fa-arrow-right"></i>
+          </Link>
+        </div>
 
-      <section id="Our-Projects-Section" className='white-text bg-secondary p-40' style={{ paddingBottom: 300 }}>
-        <h2 className='f-size-50'>Nuestros proyectos</h2>
-        <OurProjects />
-        {/* TODO: CREAR COMPONENTE DE TARJETA CON FOTO, NOMBRE Y DESCRIPCIÓN */}
-      </section>
-      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320" style={{ marginBottom: -10, marginTop: -250 }} className='mt-50'>
-        <path fill="#0077cc" fillOpacity="1" d="M0,0L16,5.3C32,11,64,21,96,48C128,75,160,117,192,128C224,139,256,117,288,128C320,139,352,181,384,170.7C416,160,448,96,480,64C512,32,544,32,576,69.3C608,107,640,181,672,208C704,235,736,213,768,197.3C800,181,832,171,864,149.3C896,128,928,96,960,112C992,128,1024,192,1056,213.3C1088,235,1120,213,1152,176C1184,139,1216,85,1248,85.3C1280,85,1312,139,1344,154.7C1376,171,1408,149,1424,138.7L1440,128L1440,320L1424,320C1408,320,1376,320,1344,320C1312,320,1280,320,1248,320C1216,320,1184,320,1152,320C1120,320,1088,320,1056,320C1024,320,992,320,960,320C928,320,896,320,864,320C832,320,800,320,768,320C736,320,704,320,672,320C640,320,608,320,576,320C544,320,512,320,480,320C448,320,416,320,384,320C352,320,320,320,288,320C256,320,224,320,192,320C160,320,128,320,96,320C64,320,32,320,16,320L0,320Z"></path>
-      </svg>
-      <section className='bg-thirt p-40 white-text'>
-        <h2 className='f-size-50'>Nuestros servicios</h2>
-        <div className='grid-c-4 gap-25 mt-50'>
-          <div>
-            <div className='center-text f-size-100'>
-              <i className="fa-solid fa-globe"></i>
+        <div className="flex space-evenly" style={{width: "80%"}}>
+          
+          {/* TODO: Hacer componentes estas etiquetas  */}
+          <div className="border-radius-30 p-10 bg-light-gray-color flex  align-center" style={{width: "180px", height:"75px"}}>
+            <div className="border-white icon-size flex align-center justify-content  border-radius-20 bg-light-gray-color mr-10">
+              <i className="fa-brands fa-github fa-xl color-primary"></i>
+            </div>
+
+            <div >
+              <p className="f-size-14">Líneas de código</p>
+              <p className="f-size-20">10K</p>
             </div>
-            <h3 className='f-size-30 mt-20'>Sitios web</h3>
-            <p className='f-size-24'>Desarrollamos sitios web a la medida de tus necesidades.</p>
           </div>
-          <div>
-            <div className='center-text f-size-100'>
-              <i className="fa-solid fa-cart-shopping"></i>
+
+          <div className="border-radius-30 p-10 bg-light-gray-color flex  align-center" style={{width: "180px", height:"75px"}}>
```

Comment:
Try not to use fiex widths and heights, it is necessary in some cases but it is not good for automatic responsive.